qcodes/instrument_drivers/tektronix/Keithley_2600_channels.py

```diff
@@ -311,33 +493,65 @@ def _fast_sweep(self, start: float, stop: float, steps: int,
                   'printbuffer(1, {}, {}.nvbuffer1.readings)'.format(steps,
                                                                      channel)]
 
-        self.write(self._parent._scriptwrapper(program=script, debug=True))
-        # we must wait for the script to execute
-        oldtimeout = self._parent.visa_handle.timeout
-        self._parent.visa_handle.timeout = 2*1000*steps*nplc/50 + 5000
+        return self._execute_lua(script, steps)
+
+    def _execute_lua(self, _script: list, steps: int) -> np.ndarray:
```

Comment:
let's make it more specific:
```suggestion
    def _execute_lua(self, _script: List[str], steps: int) -> np.ndarray:
```